poweriq-worker/src/main/java/com/vmware/flowgate/poweriqworker/jobs/PowerIQService.java

```diff
@@ -1485,20 +1485,27 @@ private String generatePduInletString(List<Inlet> pduInlets) throws JsonProcessi
       }
       return mapper.writeValueAsString(pduInletsSaveToFlowgate);
    }
+
    public Map<String,String> generatePduRateInfoMap(Pdu pdu){
 
       Map<String,String> pduRateInfo = new HashMap<String,String>();
       String rateAmps = pdu.getRatedAmps();//eg: 32A
       String ratePower = pdu.getRatedVa();//eg: 6.4-7.7kVA
       String rateVolts = pdu.getRatedVolts();//eg: 200-240V
       String rateAmpsValue = null;
+      String regEx = "([0-9]\\d*\\.?\\d*)";
```

Comment:
Will you consider the NuberUtils.isCreateable  method?  
https://commons.apache.org/proper/commons-lang/apidocs/org/apache/commons/lang3/math/NumberUtils.html#isCreatable-java.lang.String-